src/tools/styles/ToolsResult.scss

```diff
@@ -0,0 +1,15 @@
+.tools-result {
+  & .tabs__header {
+    z-index: 3;
```

Comment:
I have always set the `z-index` value within a band, that is, 100 for containers, 500 for buttons, 999 for modals. This is so if I need to put something between 100 and 200, I have got an index available. This is again nitpicking, but I thought I will mention it regardless.

Reply:
Yeah, I can put it higher. It was just what I needed to get over the nightingale track, but it definitely doesn't hurt ot have a bit of margin to put other values in between if needed